### ISSUE TITLE
Network architecture on D&D and ENZYMES

### ISSUE THREAD

**Issue by wangyuhu** [cszhangzhen/HGP-SL#5]
hello, this's a excellent paper, but i can not reproduce the previously mentioned accuracy on D&D and ENZYMES. I have read the code and found that the net arch is a 3 layers network, but according to your readme.md file, the network arch of these two dataset  is a 2 layers network arch, but i found no details of the 2 layers arch in the paper. So could you tell me the details of the 2 layers arch for  D&D and ENZYMES? Wish for your reply.

**Comment by cszhangzhen** [comment]
Hi, thanks for your interest in our work.

For the 2 layers architecture, you can simply remove the middle layer in the code.

**Comment by wangyuhu** [comment]
hi, thaks for your guidance. I have already reproduced the previously mentioned accuracy on D&D, but still could not get the same acc on ENZYMES (using the same code, just change the dataset and some hyperpara).

**Comment by cszhangzhen** [comment]
Hi,

The ENZYMES dataset is a little strange and may have a bug. Also see this issue #https://github.com/rusty1s/pytorch_geometric/issues/764. Actually, for this dataset, I evaluate the train accuracy, validation accuracy and test accuracy in each training epoch; and the best test accuracy is reported in this procedure.

**Comment by wangyuhu** [comment]
Wow, thanks very much for your prompt reply. Maybe this is really a bug in ENZYMES. Thanks again!
